Allow quartz to be used in circular saws.

luk3yx opened the pull request:
Allow quartz to be used in circular saws, fix indenting issues and make `mod.conf` 5.0-friendly.

luk3yx committed:
Allow quartz to be used in circular saws

luk3yx committed:
Fix whitespace

luk3yx committed:
Add description and depends to mod.conf.

sofar commented on init.lua:
```diff
@@ -157,34 +157,27 @@ minetest.register_abm({
-	table.insert(circular_saw.known_stairs, "quartz:block")
-	table.insert(circular_saw.known_stairs, "quartz:chiseled")
-	table.insert(circular_saw.known_stairs, "quartz:pillar")
+if minetest.global_exists("stairsplus") then
```
add `stairsplus` as optional dependency. Also should be mentioned in the readme?

luk3yx replied:
I thought `moreblocks` included `stairsplus` (it adds it as a global anyway), but I'll add `stairsplus` anyway.

luk3yx replied:
Fixed.

luk3yx committed:
Add "stairsplus" as an optional dependency.
 • Add "stairsplus" as an optional dependency.
 • Sort the optional dependencies in alphabetical order.

luk3yx committed:
Add stairsplus note into README.txt.
